phys/module_physics_init.F

```diff
@@ -96,6 +96,7 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
                          urban_map_wd,                           &
                          urban_map_gbd,                          &
                          urban_map_fbd,                          &
+                         urban_map_zgrd,                         &
```

Comment:
This additional variable needs to be in the call from start_domain_nmm.F. Optional variables are not easy in for the physics init routine. This would be consistent with having the variable available from the NMM Registry.